tests/integration/pilot/common/routing.go

```diff
@@ -1276,6 +1316,23 @@ spec:
 `, app, app, mode)
 }
 
+func useClientProtocolDestinationRule(name, app string) string {
+	return fmt.Sprintf(`apiVersion: networking.istio.io/v1beta1
+kind: DestinationRule
+metadata:
+  name: %s
+spec:
+  host: %s
+  trafficPolicy:
+    tls:
+      mode: SIMPLE
```

Comment:
is this required? a bit confused how this works with HTTP destinations?

Reply:
Not needed I think. Will cleanup